src/Discord/Discord.php

```diff
@@ -714,7 +714,7 @@ protected function handleDispatch(object $data): void
                 Event::GUILD_CREATE,
             ];
 
-            if (! $this->emittedReady && (array_search($data->t, $parse) === false)) {
+            if (! $this->emittedReady && (in_array($data->t, $parse))) {
```

Comment:
`=== false` should be `! in_array()`
```suggestion
            if (! $this->emittedReady && (! in_array($data->t, $parse))) {
```